src/styles/index.scss

```diff
@@ -1,3 +1,130 @@
+$border-color: #f3f3f3;
+$product-card-width: 200px;
+$main-fonts: 'Roboto';
+$main-text-color: #060b35;
+$second-text-color: #616070;
+$button-color: #00acdc;
+$border-radius: 5px;
+
 body {
   margin: 0;
 }
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: $product-card-width;
+  border: 1px solid $border-color;
+  border-radius: $border-radius;
+  align-items: center;
+  justify-content: center;
+
+  &__image {
+    background-image: url('../images/imac.jpeg');
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+    box-sizing: border-box;
+  }
```

Comment:
The BEM element '.product-card__image' has a margin with non-zero values, which is not recommended. BEM methodology suggests avoiding external geometry and positioning for BEM blocks and elements, such as margins. Consider refactoring this by utilizing the parent element to control the layout instead.